scripts/gene_relationships.py

```diff
@@ -254,6 +309,55 @@ def get_relationship_in_expression_space(
     return expression_dist_counts
 
 
+def get_CI_expression_relationships(
+    num_iter,
+    corr_df,
+    genes_to_consider,
+    gene_mapping_df,
+    offset_to_bin,
+    operon_df=None,
+    sum_increment=1,
+):
+    """
+    This function returns the CI for the
+    get_relationship_in_expression_space()
+    """
+
+    for i in range(num_iter):
+        # Randomly sample `genes_to_consider` with replacement
+        # This will return a sample of the same size as
+        # `genes_to_consider` but with a different random set of genes
```

Comment:
"Different" meaning in a different order? Or are you taking a subset of the genes?

Reply:
Different meaning the I am sampling from the genes_to_consider list and returning a list of genes that is the same size as the genes_to_consider where I sampled with replacement, so its a different subset of genes. I'll update this comment